samples/java-protobuf-customer-registry-kafka-quickstart/src/it/java/customer/api/CustomerActionWithKafkaIntegrationTest.java

```diff
@@ -38,7 +38,7 @@ public class CustomerActionWithKafkaIntegrationTest {
   /**
    * The test kit starts both the service container and the Kalix proxy.
    */
-  @ClassRule
+  @RegisterExtension
```

Comment:
This will behave as the `ClassRule` annotation if the class (ie `KalixTestKitResource`) implements `BeforeAllCallback` and `AfterAllCallback`.